src/neo/SmartContract/CallFlags.cs

```diff
@@ -8,12 +8,11 @@ public enum CallFlags : byte
         None = 0,
 
         ReadStates = 0b00000001,
-        WriteStates = 0b00000010,
+        WriteStates = 0b00000011,
```

Comment:
I think we'd better not to set two bits in a single flag.

Reply:
I will fix it

Reply:
The wallet should always set `ReadStates` together with `WriteStates`.

Reply:
How about renaming WriteStates to ReadWriteStates?

Reply:
The problem is if you set `ReadWriteStates` and you require `Read` the flags doesn't match